net_softnet.go

```diff
@@ -71,17 +85,59 @@ func parseSoftnet(r io.Reader) ([]SoftnetStat, error) {
 			return nil, fmt.Errorf("%d columns were detected, but at least %d were expected", width, minColumns)
 		}
 
-		// We only parse the first three columns at the moment.
-		us, err := parseHexUint32s(columns[0:3])
-		if err != nil {
-			return nil, err
+		// * Linux 2.6.23 https://elixir.bootlin.com/linux/v2.6.23/source/net/core/dev.c#L2343
+		if width == minColumns {
```

Comment:
This is probably better in a switch. Or parse only the additional column in the if clauses below.

Reply:
i've change the code to only parse additional column, 
since i got merged here https://github.com/RedHatInsights/insights-core/pull/3561 the code now support 9, 10, 11, 13 columns it should cover from linux 2.6.23 to 5.14 and later